corehq/apps/es/aggregations.py

```diff
@@ -215,6 +215,7 @@ class TermsAggregation(Aggregation):
     def __init__(self, name, field, size=None, missing=None):
         assert re.match(r'\w+$', name), \
             "Names must be valid python variable names, was {}".format(name)
+        assert size != 0, "Aggregation size must be greater than 0"
```

Comment:
Nit: Is "must be greater than 0" the actual requirement?
```suggestion
        assert size > 0, "Aggregation size must be greater than 0"
```
Or maybe the language should be changed?
```suggestion
        assert size != 0, "Aggregation size must not be zero"
```

Reply:
size can be `None` which means we can't compare it with an integer. So we can't use first suggestion.

Maybe I am overthinking it "must not be zero" would include negative numbers which are not supported. I am okay updating the language if you feel strongly about it.

Reply:
Maybe this then?
```suggestion
        assert size is None or size > 0, "Aggregation size must be greater than 0"
```

Reply:
[ae30a37](https://github.com/dimagi/commcare-hq/pull/33146/commits/ae30a37373e18393cb51058995433baf21729e6a)